nomad/structs/structs.go

```diff
@@ -10393,27 +10397,22 @@ func (a *Allocation) LastUnknown() time.Time {
 	return lastUnknown.UTC()
 }
 
-// Reconnected determines whether a reconnect event has occurred for any task
-// and whether that event occurred within the allowable duration specified by MaxClientDisconnect.
-func (a *Allocation) Reconnected() (bool, bool) {
```

Comment:
I changed the return values here because the double boolean was a bit confusing and also kind of unnecessary since there's already a [`alloc.Expired`](https://github.com/hashicorp/nomad/blob/v1.4.2/nomad/structs/structs.go#L10354-L10382) method.